src/domain/collaboration/callout/dto/callout.dto.event.message.received.ts

```diff
@@ -12,6 +12,12 @@ export class CalloutCommentsMessageReceived {
   })
   calloutID!: string;
 
+  @Field(() => String, {
```

Comment:
This should not be needed if sending a message to a Callout as there is exactly on Comments associated with a Callout, but might make the client code easier?